bootstrapper/initproto/init.proto

```diff
@@ -10,15 +10,15 @@ service API {
 
 message InitRequest {
   // repeated string autoscaling_node_groups = 1; removed
-  bytes master_secret = 2;
+  // bytes master_secret = 2; removed
   string kms_uri = 3;
   string storage_uri = 4;
-  string key_encryption_key_id = 5;
-  bool use_existing_kek = 6;
+  // string key_encryption_key_id = 5; removed
+  // bool use_existing_kek = 6; removed
```

Comment:
without double checking: maybe they are needed when using eKMS? Will check tomorrow.

Reply:
I think I saw KEK ID being part of the KMS URI. Not sure about the bool, though.

Reply:
Both removed.
However, I would like to hold off on this PR until https://github.com/edgelesssys/constellation/pull/1071 is merged

Reply:
Okay, I'll mark it as blocked for now then.

Reply:
Has been merged now. I rebased the PR. Marking it again ready for review.